src/herder/TransactionQueue.cpp

```diff
@@ -196,6 +202,14 @@ TransactionQueue::canAdd(TransactionFrameBasePtr tx,
                     return TransactionQueue::AddResult::ADD_STATUS_DUPLICATE;
                 }
 
+                // By this point, there's already a tx in the queue for this
+                // account, and only the tx with the lowest seqnum for an
+                // account can have non-zero values for these fields.
+                if (tx->getMinSeqAge() != 0 || tx->getMinSeqLedgerGap() != 0)
+                {
+                    return TransactionQueue::AddResult::ADD_STATUS_ERROR;
+                }
+
```

Comment:
I think the line below `seqNum = tx->getSeqNum() - 1;` is wrong: it should be the last tx in the sequence or the account's seqnum (if the bumped tx is the first one), otherwise we're going to check the `minSeqNum` condition based off the wrong sequence number)

Reply:
(test gaps)